x/emissions/keeper/msg_server.go

```diff
@@ -775,3 +793,153 @@ func checkNodeRegistered(ctx context.Context, ms msgServer, node sdk.AccAddress)
 	}
 	return state.ErrAddressNotRegistered
 }
+
+func (ms msgServer) ReactivateTopic(ctx context.Context, msg *state.MsgReactivateTopic) (*state.MsgReactivateTopicResponse, error) {
```

Comment:
none of these functions have unit tests and they should _all_ have unit tests before merge